lib/barcodevalidation/gtin/base.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "forwardable"
+
+module BarcodeValidation
+  module GTIN
+    class Base < BarcodeValidation::DigitSequence
+      MODULUS = 10
+
+      extend Forwardable
```

Comment:
i'm not sure where we're using forwardable in this class or its descendants?

Reply:
Nor am I, I just abstracted this class out of the main `BarcodeValidation` as-is with fe13cc9